apps/dexdrip/dexdrip.c

```diff
@@ -227,7 +227,7 @@ uint32 getSrcValue(char srcVal) {
 }
 void print_packet(Dexcom_packet* pPkt) {
     uartEnable();
-    printf("%lu %hhu %d", dex_num_decoder(pPkt->raw), pPkt->battery, adcConvertToMillivolts(adcRead(0)));
+    printf("%lu %hhu %d\n", dex_num_decoder(pPkt->raw), pPkt->battery, adcConvertToMillivolts(adcRead(0)));
```

Comment:
unfortunately this would likely cause issues with what the xdrip app itself is expecting, if you have a space before the newline it may be alright, but Im not positive